### ISSUE TITLE
Vbias0 of AD5941

### ISSUE THREAD

**Issue by wocard2** [jankae/LCR#5]
Fontend.cpp says

static constexpr uint32_t referenceVoltage = 1100000;

I think this should be Vbias0 (1.11 V, ie, 1110000) rather than the reference voltage for the DAC (1.1 V, ie, 1100000). Probably doesn't hurt and gets fixed by calibration anyway.

**Comment by jankae** [comment]
Yes, that value should have one more 1. If I remember correctly, this value doesn't affect the measurement result at all but it responsible for setting the correct DC bias for the DUT. I guess that with the current code, the DC bias has an offset of 10 mV
